code/src/components/TiviDropDownTwo.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+
+let capital = 'They'
+let lower = 'they'
+let lower2 = 'their'
+let lower3 = 'them'
+let s = ''
+let are = 'are'
+let tries = 'try'
+// let does = 'do'
+let has = 'have'
+
+let TiviDropDownTwo = ({q11Answer, q12Answer, q13Answer, q14Answer, q15Answer, q16Answer, q17Answer, q18Answer, q19Answer, q20Answer, q11Question, q12Question, q13Question, q14Question, q15Question, q16Question, q17Question, q18Question, q19Question, q20Question, gender, error}) => {
+
+    if (gender === 'Female') {
+        capital = 'She'
+        lower = 'she'
+        lower2 = 'her'
+        lower3 = 'her'
+        are = 'is'
+        tries = 'tries'
+        s = 's'
+        // does = 'does'
+        has = 'has'
+    } if (gender === 'Male') {
+        capital = 'He'
+        lower = 'he'
+        lower2 = 'his'
+        lower3 = 'him'
+        are = 'is'
+        tries = 'tries'
+        s = 's'
+        // does = 'does'
+        has = 'has'
+    } if (gender === 'Neither') {
+        capital = 'They'
+        lower = 'they'
+        lower2 = 'their'
+        lower3 = 'them'
+        are = 'are'
+        s = ''
+    }
```

Comment:
Amazed with all details you considered in this project. Great UX!

Reply:
To be honest I kind of shocked myself that I did this, but the paper describing the questionnaire said that you should provide gender-relevant questions. I wish I had used better naming conventions for the grammar though. Thank you!